index.html

```diff
@@ -249,6 +249,45 @@ <h2 class="work-title-props">Multi-Post<br/>Stories</h2>
 
             </section>
 
+            <!-- About Myself -->
+            <section class="flex-box vertical flex-center transparent-bg black-fg" id="section_1" >
+
+                <h1>About Myself</h1>
```

Comment:
- [ ] There is no space between your last project card and the about section header. Kindly address this issue 👍 
- [ ] Also the top-right corner of the about section is curved and the background color of the about page is different from the page background color. It would be great if you could add this.
- [ ] Lastly, the `About Myself` heading takes 2 lines but your heading is on 1 line. It would be nice to correct this so that your site can match the Figma template 💯 
- Your design:
![Annotation 2022-02-20 070136](https://user-images.githubusercontent.com/85113567/154831294-bc354745-06c0-45ad-b1fe-8b941e9b927d.jpg)
- Figma design:
![Annotation 2022-02-20 070157](https://user-images.githubusercontent.com/85113567/154831304-96ea5f53-909a-4b7c-9f1b-b156e5b730a5.jpg)